packages/hdwallet-ledger/src/ledger.ts

```diff
@@ -519,6 +522,16 @@ export class LedgerHDWallet
     return eth.ethGetAddress(this.transport, msg);
   }
 
+  public async solanaGetAddress(msg: core.SolanaGetAddress): Promise<string> {
+    await this.validateCurrentApp("Solana");
+    return solana.solanaGetAddress(this.transport, msg);
+  }
+
+  public async solanaSignTx(msg: core.SolanaSignTx): Promise<core.SolanaSignedTx | null> {
+    const address = await this.solanaGetAddress(msg);
+    return solana.solanaSignTx(this.transport, msg, address);
+  }
```

Comment:
ocd, add to the bottom instead of in the middle of eth functions

Reply:
246406a56b1f7445ce3872c20e21a32610221fc5